BiCo/src/app/app-routing.module.ts

```diff
@@ -36,7 +38,10 @@
     path: 'negocio-create',
     component: CrearNegocioComponent,
   },
-
+  {
+    path: 'mis-negocios',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ISPP-2021/BiCo-Frontend/pullRequest?prid=7237503)
```suggestion
    path: "mis-negocios",
```